docs/tutorials/multimodal/object_detection/detection_eval_yolov3_coco.md

```diff
@@ -0,0 +1,87 @@
+# AutoMM Detection - Evaluate Pretrained YOLOv3 on COCO Format Dataset
+:label:`sec_automm_detection_eval_yolov3_coco`
+
+In this section, our goal is to evaluate YOLOv3 model on VOC2007 dataset in COCO format. We start with yolov3 because it's extremely fast and accurate, and is a good choice to deploy with strict time and computational restrictions.
+
+To start, import MultiModalPredictor:
+
+```python
+from autogluon.multimodal import MultiModalPredictor
+```
+
+We select the YOLOv3 with MobileNetV2 as backbone, 
+for other yolov3 models see [MMDetection's YOLOV3 models](https://github.com/open-mmlab/mmdetection/tree/master/configs/yolo). 
+And we use all the GPUs (if any):
+
+```python
+checkpoint_name = "yolov3_mobilenetv2_320_300e_coco"
+num_gpus = -1  # use all GPUs
+```
+
+We create the MultiModalPredictor with selected checkpoint name and number of GPUs.
+We also need to specify the problem_type is `"object_detection"`.
+
+```python
+predictor = MultiModalPredictor(
+    hyperparameters={
+        "model.mmdet_image.checkpoint_name": checkpoint_name,
+        "env.num_gpus": num_gpus,
+    },
+    problem_type="object_detection",
+)
+```
+
+Here we use COCO17 to test. 
+See other tutorials for \[Prepare Common Public Dataset], \[Convert VOC Format Dataset to COCO Format], and \[Create Custom Dataset].
+While using COCO dataset, the input is the json annotation file of the dataset split.
+In this example, `instances_val2017.json` is the annotation file of validation split of COCO17 dataset.
+
+```python
+test_path = "coco17/annotations/instances_val2017.json"
```

Comment:
We need to provide a link or method here to let users know where he can get the json file for evaluation.

Reply:
Sure, I cite the future tutorial "Prepare COCO2017 Dataset" here. We'll also have "Prepare VOC Dataset", "Prepare Watercolor Dataset", "Convert VOC Format Dataset to COCO Format", and "Create Custom Dataset".